### ISSUE TITLE
Require cmdlet for DFW firewall status

### ISSUE THREAD

**Issue by nmbradford** [vmware/powernsx#4]
Requesting a new cmdlet for the retrieval of the firewall status. This is required to determine if the hosts have the new/updated rules and/or objects through the generation ids reported by each host.
Here is a simple example...

```
function Get-NsxFirewallStatus {
<#
.SYNOPSIS
Retrieves the specified NSX Distributed Firewall Status.

.DESCRIPTION
Enter something witty here

This cmdlet retrieves the NSX Distributed Firewall Status.

.EXAMPLE
PS C:\> Get-NsxFirewallStatus

#>

param (

    [Parameter (Mandatory=$false)]
        [string]$scopeId="globalroot-0",
    [Parameter (Mandatory=$False)]
        #PowerNSX Connection object.
        [ValidateNotNullOrEmpty()]
        [PSCustomObject]$Connection=$defaultNSXConnection

)

begin {

}

process {

    $URI = "/api/4.0/firewall/$scopeID/status"
    $response = invoke-nsxrestmethod -method "get" -uri $URI -connection $connection
    $return = $response.firewallStatus
    $return
}
}
```

**Comment by nmbradford** [comment]
Pushing to 2.2 - non critical feature request

**Comment by dcoghlan** [comment]
Fixed in #365
